### DISCUSSION TITLE
Mini Graph and Shields does Not Load

### DISCUSSION THREAD

**Discussion by c4tcom** [upptime/upptime#351] [Q&A]
Hi!
i'v been using upptime for 3 days now and i don't know why  both the shields and the Mini graphs does not load 
![pic](https://user-images.githubusercontent.com/58398485/117582959-373a2b80-b0f4-11eb-80a8-239f47a2b623.PNG)

Thanks!

**Comment by AnandChowdhary** [comment]
Can you share a link to your project?

**Reply by c4tcom** [reply]
Sure! https://github.com/adnane-X-tebbaa/DownTime-Score

**Reply by AnandChowdhary** [reply]
It looks like a 404 error somewhere, will check!
https://github.com/adnane-X-tebbaa/DownTime-Score/runs/2549403394?check_suite_focus=true#step:3:7

**Reply by c4tcom** [reply]
Awsome,Thank you!
